src/test/java/tutorspet/logic/parser/AddLessonCommandParserTest.java

```diff
@@ -47,14 +49,19 @@ public void parse_allFieldsPresent_success() {
         Lesson expectedLesson = new LessonBuilder(LESSON_FRI_8_TO_10).build();
 
         // whitespace only preamble
-        assertParseSuccess(parser, PREAMBLE_WHITESPACE + " c/1"
-                + DAY_DESC_LESSON_FRI_8_TO_10 + START_TIME_DESC_LESSON_FRI_8_TO_10
-                + END_TIME_DESC_LESSON_FRI_8_TO_10 + VENUE_DESC_LESSON_FRI_8_TO_10
+        assertParseSuccess(parser, PREAMBLE_WHITESPACE + " "
+                + PREFIX_CLASS_INDEX + "1"
+                + DAY_DESC_LESSON_FRI_8_TO_10
+                + START_TIME_DESC_LESSON_FRI_8_TO_10
+                + END_TIME_DESC_LESSON_FRI_8_TO_10
+                + VENUE_DESC_LESSON_FRI_8_TO_10
                 + NUMBER_OF_OCCURRENCES_DESC_LESSON_FRI_8_TO_10,
                 new AddLessonCommand(INDEX_FIRST_ITEM, expectedLesson));
 
         // multiple class indexes - last class index accepted
-        assertParseSuccess(parser, PREAMBLE_WHITESPACE + " c/1" + " c/1"
+        assertParseSuccess(parser, PREAMBLE_WHITESPACE + " "
+                + PREFIX_CLASS_INDEX + "1" + " "
+                + PREFIX_CLASS_INDEX + "1"
```

Comment:
Perhaps it would be good to use another class index here instead of both 1s so that it is clear the last index is accepted?